src/cmd/flux-exec.c

```diff
@@ -118,7 +118,7 @@ void completion_cb (flux_subprocess_t *p)
                 log_err_exit ("idset_create");
             (void)zhashx_insert (exitsets, buf, idset);
             if (!zhashx_freefn (exitsets, buf, idset_destroy_wrapper))
-                log_err_exit ("zhashx_freefn");
+                log_msg_exit ("zhashx_freefn");
         }
```

Comment:
Seems like this return value should be ignored, since zhashx_freefn() only returns NULL when the item doesn't exist, but we just inserted it so we know it does.

Reply:
sounds good, tweaked that last commit to ignore the return value instead.